src/lang/std/artifactGraph.ts

```diff
@@ -249,7 +249,20 @@ export function getArtifactsToUpdate({
   const cmd = command.cmd
   const returnArr: ReturnType<typeof getArtifactsToUpdate> = []
   if (!response) return returnArr
-  if (cmd.type === 'enable_sketch_mode') {
+  if (cmd.type === 'make_plane' && range[1] !== 0) {
+    // If we're calling `make_plane` and the code range doesn't end at `0`
+    // it's not a default plane, but a custom one from the offsetPlane standard library function
```

Comment:
This seems pretty hacky. Can we add a flag when we create the default planes that indicate what they are?

Reply:
I agree. I was going to say something that seems like a bit of a heuristic, which I'm not a fan of. The only reason why I didn't say anything is because I think this condition will be valid for a long time to come. Maybe add a comment on top that it's a heuristic?

Reply:
Oh yeah I just remembered that we store the IDs of the default planes in the engine command manager! I can verify that the ID is none of those; would that be concrete enough?

Reply:
Ah but that would bring a dependency into ArtifactGraph. It does kinda lead to the question: why aren't we storing the default planes in the ArtifactGraph?

Reply:
Let's make an issue for this to revisit later.

Reply:
#4504